uptime/uptime_windows.go

```diff
@@ -6,5 +6,11 @@ var getTickCount = syscall.NewLazyDLL("kernel32.dll").NewProc("GetTickCount")
 
 func get() (float64, error) {
 	r, _, err := getTickCount.Call()
+	if errno, ok := err.(syscall.Errno); ok {
+		if errno != 0 {
+			return 0, err
+		}
+		err = nil
```

Comment:
[just a comment]
According to https://golang.org/src/syscall/dll_windows.go, `err` is always non-nil, and contains `syscall.Errno`.

So I think all expected cases are covered in this `if errno, ok := err.(syscall.Errno); ok {` block, is it right?
If so, I think calculating and returning uptime can be placed here, not end of this method?